http_service/app.py

```diff
@@ -132,6 +130,39 @@ def is_running(model_name, bug_id):
     return False
 
 
+def get_bug_ids_last_change_time(bug_ids):
+    query = {"id": bug_ids, "include_fields": ["last_change_time", "id"]}
+    bug = bugzilla.get(
+        query, download_comment=False, download_attachment=False, download_history=False
+    )
+
+    return bug
+
+
+def check_bug_updated_since_last_cached(model_name, bug_id, change_time):
+    # First get the saved change time
+    change_key = change_time_key(model_name, bug_id)
+
+    saved_change_time = redis_conn.get(change_key)
+
+    # If we have no last changed time, the bug was not classified yet or the bug was classified by an old worker
+    if not saved_change_time:
+        # We can have a result without a cache time
+        if redis_conn.get(result_key(model_name, bug_id)):
+            return True
+
+        return False
+
+    return saved_change_time.decode("utf-8") != change_time
+
+
+def check_bug_updated_and_clean(model_name, bug_id, change_time):
```

Comment:
Not sure if it's better to have a function called `clean_prediction_cache`, and then in each call site do:
```
if is_prediction_invalidated(model_name, bug_id, change_time):
    clean_prediction_cache(model_name, bug_id)
```

Or maybe what you are doing now, but changing the function name to `clean_prediction_cache` (even if it's not always doing it).

I just feel the current name is too complex.